common/client-libs/gateway-client/src/socket_state.rs

```diff
@@ -98,9 +100,15 @@ impl PartiallyDelegated {
             let mut fused_receiver = notify_receiver.fuse();
```

Comment:
as a side note, I think it's now possible to remove this stupid fused streams since this method is no longer built under `wasm`